footer.inc

```diff
@@ -66,69 +66,88 @@
+    </div> 
     
-    <div class="section" id="jqueryvalidation" style="display:table; width:100%; padding:0% 10% 0% 10%">
-		<form style="width:100%" id="donateForm" method="get" action="">
-		<div class="col-project valign-wrapper" style="vertical-align: middle;">
+    <div class="container center-align" id="jqueryvalidation" style="padding:0% 5% 0% 5%;">
+		<form id="cancelForm" method="get" action="" class="center-align">
+		
+		<div class="row center-align">
+		<div class="col s12 m12 l12 center-align">
 			
-         					<div class="row" style="padding:3% 3% 3% 3%; width:100%">
-         						<div class="input-field col s12 m12 l12 center-align">
-         							<i class="material-icons prefix" style="font-size:20px;">email&nbsp;&nbsp;</i>
-          							<input placeholder="jane@gmail.com" id="email" name="email" data-error=".errorTxt1" style="font-size:20px; color:blue;" id="email" type="email">
-          							<div class="errorTxt1"></div>
+         					
+         						<div class="input-field col s12 center-align">
+          							<input placeholder="jane@gmail.com" id="cancellation_email" name="email" required data-error=".errorTxt3" style="font-size:20px; color:black;" type="email">
+          							<div class="errorTxt3" style="font-size:10px; color:red; padding:1% 0% 0% 0%"></div>
           						</div>
-							</div>
-         </div>	   
-         <div class="col-project valign-wrapper" style="vertical-align: middle; width:100%">
-        			<div class="input-field col s12 m12 l12">
-        			<button id="myBtn" class="btn-large center-align waves-effect waves-light light-blue submit" type="submit" name="action" style="width:100%;">Submit</button>
+         </div> 	
+         </div>
+         <div class="row">
+          						 <div class="input-field col s12 m6 l6">  
+                                  <input id="donationFirstName" name="firstname" placeholder="first name" type="text" required  data-error=".errorTxt1" style="font-size:20px; color:black;">
+                                  <div class="errorTxt1" style="font-size:10px; color:red; padding:2% 0% 0% 0%"></div>
+                                </div>
+                                <div class="input-field col s12 m6 l6">
+                                  <input id="donationLastName" name="lastname" placeholder="last name" type="text" required data-error=".errorTxt2" style="font-size:20px; color:black;">
+                                  <div class="errorTxt2" style="font-size:10px; color:red; padding:2% 0% 0% 0%"></div>
+                                </div>
+          						
+          						
+         </div>	  
+    
+         
+         <div class="row s12 m12 l12 center-align" style="padding:3% 0% 0% 0%;">
+        			<div class="input-field">
+        			<button id="submitBtn" class="btn-large red submit" type="submit" name="action" style="width:100%;">Submit</button>
```

Comment:
Since this is a red button, it should be more descriptive than "Submit".  Maybe "Cancel My Subscription"?

Reply:
cool. good point. i can add that.